content/en/benchmark.md

```diff
@@ -0,0 +1,128 @@
+
+### Numba
+
+Numba converts the Python files to optimized machine code during runtime. The major advantage is after compiling codes using Numba, the speed of code becomes approachable to C or FORTRAN. Numba caches all the compiled functions into a file system for future use of the same functions. We used the jit decorated functions to call another jit decorated function while implementing our code. It is advisable to add the jit decorated function to other functions too, so as to prevent it from generating slower outputs. This algorithm took 0.89 seconds for input data with 16 points, 0.05 seconds for input data with 32 points and 0.17 seconds for input data with 64 points.
+
+## Results 
+
+Table values represent the time taken by each algorithm to run, in respected datasets for 5 iterations.
+
+<html>
+<table>
+ <tr>
+  <td></td>
+  <td><b>Input 16</b></td>
+  <td><b>Input 32</b></td>
+  <td><b>Input 64</b></td>
+ </tr>
+ <tr>
+  <td><b>Pythran</b></td>
+  <td>10.37</td>
+  <td>41.88</td>
+  <td>160.15</td>
+ </td>
+ <tr>
+  <td><b>Numba</b></td>
+  <td>0.89</td>
+  <td>0.05</td>
+  <td>o.17</td>
+ </tr>
+ <tr>
+  <td><b>Pythran Naive</b></td>
+  <td>0.01</td>
+  <td>0.05</td>
+  <td>0.17</td>
+ </tr>
+ <tr>
+  <td><b>NumPy</b></td>
+  <td>10.15</td>
+  <td>45.2</td>
+  <td>161.01</td>
+ </tr>
+ <tr>
+  <td><b>Pure NumPy</b></td>
+  <td>10.56</td>
+  <td>41.59</td>
+  <td>169.4</td>
+ </tr>
+</table>
+</html>
+
+## Code Folders
+
+* These codes are originally taken from [here](https://github.com/paugier/nbabel)
+* Benchmark code: [here](https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/benchmark-2.py)
+* Graph: [here](https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/plot-modified-1.py)
+
+<html>
+ <table>
+<tr> 
+  <td><b>Algorithms</b></td>
+  <td><b>Source Codes</b></td>
+</tr>
+<tr>
+  <td>Pythran</td>
+  <td> <a href = "https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/bench.py">bench.py</a></td>
+</tr>
+<tr> 
+  <td>Numba</td>
+  <td> <a href = "https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/bench_numba.py">bench_numba.py</a></td>
+</tr>
+<tr>
+  <td>Pythran Naive</td>
+  <td><a href = "https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/bench_numpy_highlevel_jit.py">bench_numpy_highlevel_jit.py</a></td>
+</tr>
+<tr>
+  <td>NumPy</td>
+  <td><a href = "https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/bench_numpy_highlevel.py">bench_numpy_highlevel.py</a></td>
+</tr>
+<tr>
+  <td>Pure NumPy</td>
+  <td><a href = "https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/bench_numpy_highlevel.py">bench_pure_numpy_highlevel.py</a></td>
+</tr>
+  </table>
+</html>
+
+
+## Conclusion
+
+* These benchmarks are run with an Intel Core i7 processor.
```

Comment:
This probably belongs in the caption of the graph. With a bit more details about the exact CPU. For example:
```
$ cat /proc/cpuinfo | grep Intel
```
And then slightly reformat like: `Intel i9-7920X @ 2.90GHz`

Reply:
Apologize, I will update that. Thanks!